BSP/STM32F413/Src/BSP_ADC.c

```diff
@@ -0,0 +1,50 @@
+ /* Program for UTSVT BeVolt's Battery Protection System*/
+ /* Copyright (c) 2021 UT Longhorn Racing Solar */
+#include "BSP_ADC.h"
+
+void BSP_ADC_Init(){
```

Comment:
This function should have a comment

Reply:
@tasniabari after you add a comment for this function to BSP_ADC.h, can you copy-paste the function comment here?